test-integration/helpers/user.coffee

```diff
@@ -60,6 +63,13 @@ class User extends TestHelper
         @logInLocal(username)
       else
         @logInDeployed(username, password)
+    # Inject a little CSS to unfix the pesky fixed navbar
+    .then =>
+      @test.driver.executeScript ->
+        hider = document.createElement('style')
+        hider.textContent = '.navbar-fixed-bottom, .navbar-fixed-top { position: initial !important; } body { padding: 0; }'
+        document.head.appendChild(hider)
+
 
```

Comment:
yep, I can remove the `scrollTop` & see what happens. This probably needs to also include styling for other fixed elements on the page (like the bottom bar when doing an assignment)